src/app/common/ResourceCodeBlock.vue

```diff
@@ -117,9 +107,7 @@ const fetcher = async (_params?: SingleResourceParameters): Promise<Entity> => {
 }
 
 async function getYamlAsKubernetes() {
-  const resourceKubernetes = await (typeof props.resourceFetcher !== 'undefined' ? props.resourceFetcher({ format: 'kubernetes' }) : fetcher())
-
-  return toYamlRepresentation(resourceKubernetes)
```

Comment:
All of the `resourceFetcher` stuff here is the old stuff which was kept in so we could support both implementations at once (both old and new). Now we have rolled everything out, we don't need this older  approach.